packages/wonder-blocks-core/util/util.js

```diff
@@ -56,7 +56,14 @@ export function processStyleList<T: Object>(
                 // It doesn't accept kebab-case in media queries and instead
                 // prefers camelCase.
                 for (const [key, value] of Object.entries(child._definition)) {
-                    def[key.replace("-width", "Width")] = value;
+                    // This regex converts all instances of -{lowercaseLetter}
+                    // to the uppercase version of that letter, without the
+                    // leading dash.
+                    def[
+                        key.replace(/-[a-z]/g, (match) =>
+                            match[1].toUpperCase(),
```

Comment:
I don't understand why a group isn't necessary here... does the stuff inside `[ ]` count as a group.

Reply:
The `[ ]` doesn't count as a group, but a group isn't necessary because there's no additional operations on that group.

Reply:
Ah... it's grabbing the first character from the `match`.  That took me way too long to figure out. 😞